realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -1093,7 +1093,7 @@ public void run() {
                     transaction.execute(bgRealm);
 
                     if (!Thread.currentThread().isInterrupted()) {
-                        bgRealm.commitTransaction(new Runnable() {
+                        bgRealm.commitTransaction(true, new Runnable() {
```

Comment:
`true` here is actually redundant. Here, the realm is supposed to be closed before sending notifications, which means, even true is passed here, the local realm won't be notified because of it is supposed to be closed before.